.pre-commit-config.yaml

```diff
@@ -150,6 +150,10 @@ repos:
             cpp/src/io/parquet/ipc/Message_generated[.]h$|
             cpp/src/io/parquet/ipc/Schema_generated[.]h$
           )
+  - repo: https://github.com/DavidAnson/markdownlint-cli2
```

Comment:
I think this library has the ability to enforce one sentence per line, but we can use something else too.